package JAVA.EXCEPTION;

public class nested {
    int []a=new int[2];
    try{
        try{
    int a[3]=5;}
    catch()

    
}
